gcs/src/test/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemIntegrationTest.java

```diff
@@ -1312,6 +1312,18 @@ public void xattr_statistics() throws IOException {
     assertThat(myGhfs.delete(testRoot, true)).isTrue();
   }
 
+  public void http_IOstatistics() throws IOException {
+    GoogleHadoopFileSystem myGhfs = createInMemoryGoogleHadoopFileSystem();
+
+    FSDataOutputStream fout = myGhfs.create(new Path("/file1"));
+    fout.writeBytes("Test Content");
+    fout.close();
+
+    assertThat(myGhfs.getIOStatistics().counters().get(INVOCATION_CREATE.getSymbol())).isEqualTo(1);
+    assertThat(myGhfs.getIOStatistics().counters().get("action_http_get_request")).isEqualTo(3);
```

Comment:
Please add a comment on what the exact requests are ?

I suspect 3 get calls might be an issue, I don't see a read call here.

Reply:
Please add a test to capture the failure statistics as well